packages/core/src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -73,12 +73,14 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
 
     private _renderButton(containerId: string, methodId: string, paypalcommerce: PaypalCommerceButtonInitializeOptions): void {
         const { buyNowInitializeOptions, initializesOnCheckoutPage, style, onComplete } = paypalcommerce;
+
+        const paypalCommerceSdk = this._getPayPalCommerceSdkOrThrow();
+
         const state = this._store.getState();
         const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(methodId);
         const { isHostedCheckoutEnabled } = paymentMethod.initializationData;
-        const paypalCommerceSdk = this._getPayPalCommerceSdkOrThrow();
```

Comment:
All look good.
Just want to check, did `prettier` make this line change? 
I'm confused why does `const paypalCommerceSdk = this._getPayPalCommerceSdkOrThrow();` stand out?

Reply:
I moved this line to the place before getting any data from state because it is not necessary to trigger state if the PayPalSdk is not defined.